Updating Existing code

Grasiyan opened the pull request:
Updating existing codes as per layout option 2 and optimising class names

Grasiyan committed:
Updating Existing code

Updating existing codes as per layout option 2 and optimising class names

devstack013 commented on assets/css/index.css:
```diff
@@ -169,3 +183,90 @@ body {
+}
+
+.top-devs-heading {
+  padding: 10px 0px 10px 5%;
```
Why padding in percent

devstack013 commented on assets/css/index.css:
```diff
@@ -169,3 +183,90 @@ body {
+}
+
+.top-devs-card {
+  width: 10%;
```
Do not give width in 10%

Grasiyan committed:
Merging code into index

Merging all code to index.html and removed unnecessery classes, updated classes with proper names
